st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -495,9 +508,9 @@ def format_output_filename(cmd):
         :rtype: ``str``
         """
         if six.PY3:
-            cmd = cmd.translate(cmd.maketrans('', '', r""" !@#$%^&*()[]{};:,./<>?\|`~=+"'"""))
+            cmd = cmd.translate(cmd.maketrans('', '', """ !@#$%^&*()[]{};:,./<>?\|`~=+"'"""))
```

Comment:
dropping the preceding `r` here changes the treatment of the `\` character.  It's now treated as an escape character instead of a string literal.   Was that intentional ?

Reply:
Mystified.  I pulled in the original st2/v2.10.4 changes.  Perhaps my finger stuck on the r key.  I'll fix this and remove the s3 url in this file tomorrow.

Reply:
I thought I had written here already, but I fixed the above error last night and pushed into this branch.  I'd like to get this merged into the plexxi-v2.10.4 branch soon.